From: Michel Normand <normand@linux.vnet.ibm.com>
Subject: qtwebkit 23 ppc64le
Date: Fri, 09 May 2014 18:19:56 +0200

qtwebkit ppc64le do not define WTF_CPU_BIG_ENDIAN.
continue to use WTF_CPU_PPC64 and do not define a new WTF_CPU_PPC64LE.

Signed-off-by: Michel Normand <normand@linux.vnet.ibm.com>
---
 Source/WTF/wtf/Platform.h |    2 ++
 1 file changed, 2 insertions(+)

Index: webkit-qtwebkit-23/Source/WTF/wtf/Platform.h
===================================================================
--- webkit-qtwebkit-23.orig/Source/WTF/wtf/Platform.h
+++ webkit-qtwebkit-23/Source/WTF/wtf/Platform.h
@@ -110,8 +110,10 @@
 #if   defined(__ppc64__) \
     || defined(__PPC64__)
 #define WTF_CPU_PPC64 1
+#if !defined(__LITTLE_ENDIAN__)
 #define WTF_CPU_BIG_ENDIAN 1
 #endif
+#endif
 
 /* CPU(SH4) - SuperH SH-4 */
 #if defined(__SH4__)
